src/riakc_ts_get_operator.erl

```diff
@@ -46,6 +46,10 @@ deserialize({error, {Code, Message}}) when is_integer(Code), is_atom(Message) ->
     {error, {Code, iolist_to_binary(atom_to_list(Message))}};
 deserialize({error, Message}) ->
     {error, Message};
+deserialize(tsgetresp) ->
+    {ok, {[], []}};
+deserialize({tsgetresp, {_, _, []}}) ->
+    {[], []};
```

Comment:
We want all of these to return `ok`, right? In which case this clause shouldn't be here. @hmmr already has the other change in #277